src/Tizen.NUI/src/public/Layouting/GridLayout.cs

```diff
@@ -252,216 +362,104 @@ protected override void OnMeasure(MeasureSpecification widthMeasureSpec, Measure
-                    _totalHeight = Math.Min(_totalHeight, heightSize);
-                    _totalHeight = Math.Max(_totalHeight, (int)SuggestedMinimumHeight.AsRoundedValue());
-
-                    heightSize = _totalHeight;
-                } // Child exists
-
-                // In the case of AT_MOST, availableContentHeight is the max limit.
-                availableContentHeight = heightSize - gridLayoutPadding.Top - gridLayoutPadding.Bottom;
-            }
-            else
-            {
-                // Grid expands to fit content
-                // If number of columns AUTO_FIT then set to 1 column.
-                _columns = (_columns > 0) ? _columns : 1;
-                // Calculate numbers of rows, round down result as later check for remainder.
-                _rows = childCount / _columns;
-                // If number of cells not cleanly dividable by columns, add another row to house remainder cells.
-                _rows += (childCount % _columns > 0) ? 1 : 0;
-
-                heightSize = desiredChildHeight * _rows + gridLayoutPadding.Top + gridLayoutPadding.Bottom;
-                availableContentHeight = heightSize - gridLayoutPadding.Top - gridLayoutPadding.Bottom;
-            }
-            // If number of columns not defined
-            DetermineNumberOfColumns(availableContentWidth);
-
-            // Locations define the start, end,top and bottom of each cell.
-            _locations.CalculateLocations(_columns, availableContentWidth, availableContentHeight, childCount);
-
-            return availableContentHeight;
+            /// <summary>
+            /// Respect mesured size of the child.
+            /// </summary>
+            None = 0,
+            /// <summary>
+            /// Resize to completely fill the space.
+            /// </summary>
+            Fill = 1,
+            /// <summary>
+            /// Expand to share available space in GridLayout.
+            /// </summary>
+            Expand = 2,
+            /// <summary>
+            /// Expand to share available space in GridLayout and fill the space.
+            /// </summary>
+            ExpandAndFill = Fill + Expand,
         }
 
         /// <summary>
-        /// Assign a size and position to each of its children.<br />
+        /// The alignment of the grid layout child.
         /// </summary>
-        /// <param name="changed">This is a new size or position for this layout.</param>
-        /// <param name="left">Left position, relative to parent.</param>
-        /// <param name="top"> Top position, relative to parent.</param>
-        /// <param name="right">Right position, relative to parent.</param>
-        /// <param name="bottom">Bottom position, relative to parent.</param>
-        /// <since_tizen> 6 </since_tizen>
-        protected override void OnLayout(bool changed, LayoutLength left, LayoutLength top, LayoutLength right, LayoutLength bottom)
+        public enum Alignment
```

Comment:
Need to add [EditorBrowsable(EditorBrowsableState.Never)] before ACR.
It should not have ACR Require tag.

Please check others too.